docs/02.API-REFERENCE.md

```diff
@@ -430,6 +430,18 @@ typedef struct jerry_context_t jerry_context_t;
 
 *New in version 2.0*.
 
+## jerry_container_operation_t
+
+Enum that contains the supported container operation types
+  - JERRY_CONTAINER_OP_ADD - Set/WeakSet add operation
+  - JERRY_CONTAINER_OP_GET - Map/WeakMap get operation
+  - JERRY_CONTAINER_OP_SET - Map/WeakMap set operation
+  - JERRY_CONTAINER_OP_HAS - Set/WeakSet/Map/WeakMap has operation
+  - JERRY_CONTAINER_OP_DELETE - Set/WeakSet/Map/WeakMap delete operation
+  - JERRY_CONTAINER_OP_SIZE - Set/WeakSet/Map/WeakMap size operation
+  - JERRY_CONTAINER_OP_CLEAR - Set/Map clear operation
```

Comment:
We should somehow/somewhere describe that for each operation what should be arguments and arguments number. Maybe at the method's description.